provisioning/azure-devops-deploy.sh

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
 set -Eeuo pipefail
 
+export COMMIT=$(echo "$BUILD_BUILDID" | cut -d'-' -f 2)
+
 # Checkout correct commit
-if [ "$RELEASE_ARTIFACTS__KEBOOLA_KEBOOLA_AS_CODE_SOURCEBRANCH" == "master" ]; then
-  COMMIT=$(echo "$BUILD_BUILDID" | cut -d'-' -f 2)
+if [ "$RELEASE_ARTIFACTS__KEBOOLA_KEBOOLA_AS_CODE_SOURCEBRANCH" == "main" ]; then
```

Comment:
To je imho bug? V tomhle repu není `master` branch.

Reply:
Nechceš tohle nechat na vlastní PR? Aby to nevyvedlo něco nečekaného, je tam ten git checkout.

Reply:
ok!

Reply:
hmm pozeram ze ta premenna `$COMMIT` sa v kode nepouziva alebo je to v pipeline?
https://github.com/keboola/keboola-as-code/search?p=3&q=%22%24COMMIT%22

Reply:
původně se použila jen tady o řádek níž: https://github.com/keboola/keboola-as-code/blob/af37b676dd6015e8ea9c77c78c16d26b439609d4/provisioning/azure-devops-deploy.sh#L9-L9

Reply:
aha som slepy :(